packages/wallet/frontend/src/ui/forms/Input.tsx

```diff
@@ -40,9 +40,14 @@ export const Input = forwardRef<HTMLInputElement, InputProps>(
     const id = useId()
     return (
       <div>
-        <div className="mt-1">
+        <div
+          className={cx(
+            'mt-1',
+            type === 'checkbox' && 'flex flex-row-reverse justify-evenly'
+          )}
+        >
           {label && (
-            <Label htmlFor={id} hint={labelHint}>
+            <Label htmlFor={id} hint={labelHint} inputType={type}>
```

Comment:
Instead of doing this into one single comment, I think it's better if he have a separate `Checkbox component`.